meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -80,14 +84,75 @@ function filterLayerMappings(
 	return result
 }
 
+function getFilteredTableValuesFromComparison(
+	targetFullAttribute: string,
+	item: ConfigManifestEntrySelectFromTableEntryWithComparisonMappings<boolean>,
+	configPath: string,
+	showStyleBaseDBObject: ShowStyleBase,
+	showStyleVariantDBObject?: ShowStyleVariant
```

Comment:
I don't think we currently use `SELECT_FROM_COLUMN` Studio  either (not completely sure though), but in any case you are right since this need could arise. Do you have any suggestions as to what i could call it instead?

Reply:
Updated!